### PR TITLE
Updating the website to reflect the switch to workspaces.openshift.com / Developer Sandbox

### PR DESCRIPTION
**Pull request by ibuziuk**
Related issue - https://issues.redhat.com/browse/RHDEVDOCS-2508

Try it on Hosted Che
[Contribute](https://che.openshift.io/f?url=https://github.com/ibuziuk/che-website/tree/RHDEVDOCS-2508)

image

### CODE REVIEW

File: che/codeready-contribute.svg [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (benoitf) [inline che/codeready-contribute.svg:1]: on che website it shouldn't have any codeready images

Comment (ibuziuk) [reply che/codeready-contribute.svg:1]: @benoitf right, but I was thinking if we could host the image for contributions here (now I think that it is not a good idea)? if it is not the case, I believe the better option would be https://github.com/ibuziuk/codeready-workspaces/tree/crw-2-rhel-8/assembly/branding

Comment (ibuziuk) [reply che/codeready-contribute.svg:1]: @benoitf opened - https://github.com/redhat-developer/codeready-workspaces/pull/441

**Comment by ibuziuk** [comment]
@l0rd @benoitf PR is ready, plan to merge it on March 1st

**Review by benoitf — APPROVED** [review]

**Review by l0rd — APPROVED** [review]

**Pull request merged**